airflow/providers/google/suite/operators/drive.py

```diff
@@ -0,0 +1,115 @@
+    The local files can be deleted after upload (optional)
+
+    :param local_paths: Python list of local file path strings
+    :param drive_folder: path of the Drive folder
+    :param gcp_conn_id: Airflow Connection ID for GCP
+    :param delete: should the local files be deleted after upload?
+    :param chunk_size: File will be uploaded in chunks of this many bytes. Only
+        used if resumable=True. Pass in a value of -1 if the file is to be
+        uploaded as a single chunk. Note that Google App Engine has a 5MB limit
+        on request size, so you should never set your chunk size larger than 5MB,
+        or to -1.
+    :param resumable: True if this is a resumable upload. False means upload
+        in a single request.
+    :param delegate_to: The account to impersonate using domain-wide delegation of authority,
+        if any. For this to work, the service account making the request must have
+        domain-wide delegation enabled.
+    :param impersonation_chain: Optional service account to impersonate using short-term
+        credentials, or chained list of accounts required to get the access_token
+        of the last account in the list, which will be impersonated in the request.
+        If set as a string, the account must grant the originating account
+        the Service Account Token Creator IAM role.
+        If set as a sequence, the identities from the list must grant
+        Service Account Token Creator IAM role to the directly preceding identity, with first
+        account from the list granting this role to the originating account
+    :return: Remote file ids after upload
+    :rtype: Sequence[str]
+    """
+
+    template_fields = (
+        'local_paths',
+        'drive_folder',
+    )
+
+    def __init__(
+        self,
+        local_paths,
+        drive_folder: str,
+        gcp_conn_id: str,
+        delete: bool = False,
+        chunk_size: int = 100 * 1024 * 1024,
+        resumable: bool = False,
+        delegate_to: Optional[str] = None,
+        impersonation_chain: Optional[Union[str, Sequence[str]]] = None,
+        **kwargs,
+    ) -> None:
+        super().__init__(**kwargs)
+        self.local_paths = local_paths
+        self.drive_folder = drive_folder
+        self.gcp_conn_id = gcp_conn_id
+        self.delete = delete
+        self.chunk_size = chunk_size
+        self.resumable = resumable
+        self.delegate_to = delegate_to
+        self.impersonation_chain = impersonation_chain
+
+    def execute(self, context: Any) -> Sequence[str]:
+        hook = GoogleDriveHook(
+            gcp_conn_id=self.gcp_conn_id,
+            delegate_to=self.delegate_to,
+            impersonation_chain=self.impersonation_chain,
+        )
+
+        remote_file_ids = []
+
+        for local_path in self.local_paths:
+            self.log.info(f'Uploading file to Google Drive: {local_path}')
+            file_name = local_path.split('/')[-1]
```

Comment:
I would suggest using `Path` from [pathlib](https://docs.python.org/3/library/pathlib.html) so you can do `local_path.name` instead of splitting string.

Reply:
How about changing both local_paths and drive_folder to use `Path`?